CMakeLists.txt

```diff
@@ -32,6 +32,11 @@ include(ExternalDependenciesVersions)
 include(LoadFetchContent)
 include(AddCustomTargetSubproject)
 
+########################################
+#### user-defined configuration options
+########################################
+option(TTG_PARSEC_USE_BOOST_SERIALIZATION "Whether to select Boost serialization methods in PaRSEC backend" ON)
```

Comment:
What is the use of this option? Is there a similar option for MADNESS? Isn't support for madness/boost/cereal transparent to the backend?

Reply:
That was the consensus... But the first serialization does not write in memory, so costs way less, and if we're lucky the serialized object fits in some levels of cache, so the sequence serialize_to_get_the_size() ; allocate() ; serialize() may not cost significantly more than serialize() by itself. Alternative is to work with re-dimensionable / re-allocatable objects which is not too good either...